### DISCUSSION TITLE
Welcome to KeepingYouAwake Discussions!

### DISCUSSION THREAD

**Discussion by newmarcel** [newmarcel/KeepingYouAwake#166] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by norskedalen** [comment]
Hey @newmarcel, thank you for building and maintaining this awesome tool! My work laptop is locked down by 3rd party software that automatically updates and applies MacOS Profiles that lock down settings like the lock screen sleep timer, so even though I have an Admin user and tried commands like `sudo defaults -currentHost write com.apple.screensaver idleTime 0` it wouldn't work. With your tool though I can easily get around this super annoying issue. Thanks!!!
